v2/blockstore/readwrite.go

```diff
@@ -59,50 +66,164 @@ func WithIndexPadding(p uint64) Option {
 // This can help avoid redundancy in a CARv1's list of CID-Block pairs.
 //
 // Note that this compares whole CIDs, not just multihashes.
-func WithCidDeduplication(b *ReadWrite) {
+func WithCidDeduplication(b *ReadWrite) { // TODO should this take a bool and return an option to allow disabling dedupliation?
```

Comment:
the current is setup (default off, option for on) seems good enough for the first pass

Reply:
In the context of finalising the interfaces, do you think we should keep this function as is, instead of taking a `bool` for the v2 release? If released we need to keep it for the lifetime of v2 and introduce a `WithCidDeduplicationDisabled` or something like that if we want to allow disabling of deduplication.

On a side note, other option functions in this file return `Option` so whichever we decide it would be good to keep them consistent.

Reply:
being consistent with an `Option` seems good. i don't think i have strong feelings about whether there's a boolean argument